lib/ide/ast/impl/src/assoc.rs

```diff
@@ -0,0 +1,85 @@
+//! Rules for describing operator associativity.
+//!
+//! NOTE: They should be kept in sync with enso's implementation at:
+//! `enso/Syntax/definition/src/main/scala/org/enso/syntax/text/ast/opr/Assoc.scala`
+
+
+use crate::prelude::*;
+
+use lazy_static::lazy_static;
+use regex::Regex;
+
+/// Operator associativity.
+#[derive(Clone,Copy,Debug,PartialEq)]
+pub enum Assoc {
+    /// Operator left-associative, i.e. a~b~c is (a~b)~c
+    Left,
+    /// Operator right-associative, i.e. a~b~c is a~(b~c)
+    Right
+}
```

Comment:
remove doc comments. In trivial cases we prefer to use `allow(missing_docs)` than just make the code unreadable.